advanced/Scripts/piholeDebug.sh

```diff
@@ -618,7 +618,7 @@ detect_ip_addresses() {
         # Local iterator
         local i
         # Display the protocol and interface
-        log_write "${TICK} IPv${protocol} address(es) bound to the ${PIHOLE_INTERFACE} interface:"
+        log_write "${TICK} IPv${protocol} address(es) available to the ${PIHOLE_INTERFACE} interface:"
```

Comment:
```suggestion
        log_write "${TICK} IPv${protocol} address(es) attached to the ${PIHOLE_INTERFACE} interface:"
```

Reply:
Bound is a common term.  Apps bind to ports and interfaces.